main.go

```diff
@@ -29,6 +30,26 @@ func main() {
 		os.Exit(1)
 	}
 
+	if params.suppressions != "" {
+		file, err := os.Open(params.suppressions)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, "Cannot open provided suppression file: ", err)
+			os.Exit(1)
```

Comment:
We _might_ be outgrowing this early-out pattern - perhaps we should have something like:
```golang
func main() {
    if err := app.run(); err != nil {
        // print error
        os.Exit(1)
     }
}
```

Reply:
Sure, I'd be fine with that.